### PR TITLE
Us102 nika

### PR DESCRIPTION
**Pull request by VeronikaKrasnokutska**
Check the code, please

### CODE REVIEW

File: src/us100/us102/us102_FeatureA.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package us100.us102;
+
+public class us102_FeatureA {
+//test1
+
+    //test2
+
```
Comment (BadamkhandB) [inline src/us100/us102/us102_FeatureA.java:7]: how did you test 2

Comment (tsjuder1) [reply src/us100/us102/us102_FeatureA.java:7]: The code looks amazing!

Comment (tsjuder1) [reply src/us100/us102/us102_FeatureA.java:7]: there has to be another code here

**Review by tsjuder1 — CHANGES_REQUESTED** [review]
check line 2

**Review by tsjuder1 — APPROVED** [review]
I guess it's better

**Pull request merged**